clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -3637,3 +3637,28 @@ void tools::setComplexRange(const Driver &D, StringRef NewOpt,
   LastOpt = NewOpt;
   Range = NewRange;
 }
+
+void tools::constructLlvmLinkCommand(Compilation &C, const Tool &T,
```

Comment:
I find `Llvm` really annoying…could you change it to `LLVM`?

Reply:
Done, thanks